example_test.go

```diff
@@ -342,6 +342,35 @@ func ExampleClient_UndoFollowing() {
 	fmt.Println(uf)
 }
 
+func ExampleClient_Blocking() {
+	client := gotwtr.New("key")
+	b, err := client.Blocking(context.Background(), "user_id")
```

Comment:
The way this parameter is passed is different from the other examples, but I did it this way because it's hard to tell which id the id refers to.

Reply:
Ah, what does it mean ?
I haven't catch your thought, yet.
Please describe more detail.

Reply:
Other example code client method func (e.g client.Blocking) passed `id` parameter, but only `id` can't understand what refers to. So i think `user_id` , `target_user_id` , `source_user_id` and etc is better for users.`

Reply:
Ah, I agree with you. Go ahead 😄